packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -145,7 +145,8 @@ const SignInVerificationMachine = setup({
   },
   guards: {
     isResendable: ({ context }) => context.resendable || context.resendableAfter === 0,
-    isNeverResendable: ({ context }) => context.currentFactor?.strategy === 'password',
+    isNeverResendable: ({ context }) =>
+      context.currentFactor?.strategy === 'password' || context.currentFactor?.strategy === 'passkey',
```

Comment:
Consider consolidating this logic and the `nonPreparable` logic so we don't have to maintain separate checks.